meshtastic/module_config.proto

```diff
@@ -183,6 +183,13 @@ message ModuleConfig {
      * Only applicable if the board uses pull-up resistors on the pin
      */
     bool use_pullup = 8;
+
+    /*
+     * Whether or not all the state transition should be broadcasted.
+     * Causes an additional message to be sent to mesh when detection state goes
+     * back to false.
+     */
+    bool broadcast_all_state_transitions = 9;
   }
```

Comment:
Can we make it an ENUM instead? Per Discussion on discord in #use-cases, this way we could be more flexible here.

```
logic high
logic low
rising edge
falling edge
either edge
```
and probably more i just forgot ...